framework/web_app/src/pages/ProductSearch.vue

```diff
@@ -11,7 +11,7 @@
 
     <div v-show="showFiltersContainer" class="no-wrap row q-pa-sm scroll scrollbar-none">
 
-        <select-component label="Stores" :multiple="true" :options="merchantStore.merchants"
+        <select-component label="Stores" :multiple="true" :options="merchantStore.merchants.filter(m => m.is_enabled)"
```

Comment:
Although a small dataset this filter is going to apply every time this page or component is rendered.

Would recommend creating a computed method in this file that is getEnabledMerchants or getEnabledStores.

Reply:
fixed :D using method from store now